test/source/win/_posts/2009-05-24-yaml-linebreak.markdown

```diff
@@ -1,7 +1,7 @@
----
-layout: post
-title: "Test title"
-tag: "Ruby"
----
-
+---
+layout: post
+title: "Test title"
+tag: "Ruby"
+---
+
 This is the content
```

Comment:
dunno.. I copied the file from master branch and overwrote the file on this branch.. still no success..
perhaps if I rebase to upstream........